src/bors/handlers/mod.rs

```diff
@@ -324,4 +329,15 @@ mod tests {
         })
         .await;
     }
+
+    #[tracing_test::traced_test]
+    #[sqlx::test]
+    async fn pr_fetch_error(pool: sqlx::PgPool) {
```

Comment:
```suggestion
    async fn do_not_load_pr_on_unrelated_comment(pool: sqlx::PgPool) {
```
Something like this would make it easier to understand what's the test designed for.